Jenkinsfile

```diff
@@ -182,8 +191,43 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
         }
     }
 }
-//node('boardfarm') {
-    stage('Intergration test') {
-        //TODO run the boardfarm test
+node('boardfarm') {
+    stage('Integration test') {
+        deleteDir()
+
+        unarchive mapping: ['bin/pistachio/*.ubi': '.']
+        sh 'cp bin/pistachio/$(ls bin/pistachio | egrep "*\\.ubi") ${WEBSERVER_PATH}/image.ubi'
+
+        sh 'sshpass -p "root" scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null \
+        ${BOARDFARM_DIRECTORY}/ota_update.sh ${BOARDFARM_DIRECTORY}/ota_verify.sh root@${WAN_IP}:~/ || exit 1'
```

Comment:
Why the 'exit 1' needed? Surely it will have a non-zero return code in those cases anyway?